src/main/java/assignment1/models/ImportedItem.java

```diff
@@ -0,0 +1,47 @@
+package assignment1.models;
+
+import assignment1.exceptions.InvalidArguementsException;
+import assignment1.models.enums.ItemType;
+import assignment1.utils.ValidateType;
+
+public class ImportedItem extends Item {
```

Comment:
Item suffix is not required in child class name. It should be Imported, or Raw etc

Reply:
Resolved.